iroh/src/node/rpc/docs.rs

```diff
@@ -111,18 +110,21 @@ impl DocsEngine {
         Ok(DropResponse {})
     }
 
-    pub fn doc_list(&self, _req: DocListRequest) -> impl Stream<Item = RpcResult<DocListResponse>> {
-        let (tx, rx) = flume::bounded(ITER_CHANNEL_CAP);
+    pub fn doc_list(
+        &self,
+        _req: DocListRequest,
+    ) -> impl Stream<Item = RpcResult<DocListResponse>> + Unpin {
```

Comment:
Added Unpin here. it was unpin before - implicitly assumed, but it is better to be explicit. Maybe we should remove this later, but for now let's keep it as close as possible to before. Otherwise I think this would be a breaking change.